### PR TITLE
[T4A3][W13-B4] Conan Kian

### PR DESCRIPTION
**Pull request by ckjr**
Ready for review

### CODE REVIEW

**Comment by ckjr** [comment]
Review in class

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:2]: Please include a header comment to indicate what this class/method does?

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+public class Contact {
+    protected boolean isPrivate;
+    
+    public boolean isPrivate() {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:6]: you can move the value also here, no?

**Comment by okkhoy** [comment]
@ckjr think of all methods and members that you can move up to the parent class.
Some comments are added to the code, pls take a look, ack and close the PR when you have read them.

**Comment by ckjr** [comment]
Acknowledged

**Pull request closed without merging**